src/main/java/busTickets/FindMaxViolation.java

```diff
@@ -0,0 +1,21 @@
+package busTickets;
+
+public class FindMaxViolation {
+    public static String findMaxViolation(int a, int b, int c) {
+        int max = a;
+        if (b > max) {
+            max = b;
+        }
+        if (c > max) {
+            max = c;
+        }
+
+        if (a == max) {
+            return "Start date";
```

Comment:
don't use 'magic' values, create separate variables/enum for this

Reply:
I hope i resolved it by creating variables of string type in "findMaxViolation" so Garbage collector could take care of them after the execution of method